dbt/adapters/spark/impl.py

```diff
@@ -132,38 +134,79 @@ def add_schema_to_cache(self, schema) -> str:
         # so jinja doesn't render things
         return ""
 
+    def parse_information(self, table_name):
+        information = ""
+        kwargs = {"table_name": table_name}
```

Comment:
Is there any benefit to doing this versus embedding this in the method call on line 141?

Reply:
Nice one, I agree that it is better to inline it in the method call.